controllers/AuthController.js

```diff
@@ -78,5 +78,94 @@ class AuthController {
       }
     })(req, res, next);
   }
+
+  /**
+   * @author Caleb
+   * @param {Object} req
+   * @param {Object} res
+   * @param {*} next
+   * @returns {Object} Returns the response
+   */
+  static async forgotPassword(req, res) {
+    const {
+      body: { user }
+    } = req;
+
+    try {
+      const reset = await User.findOne({
+        where: { email: user.email, confirmed: 'confirmed' },
+        attributes: ['id', 'email']
+      });
+      if (!reset) {
+        return res
+          .status(404)
+          .json({ status: 404, message: 'No user found with that email address' });
+      }
+      const { id, email } = reset.get();
+      const createReset = await ResetPassword.create({ userId: id });
+      const { resetCode } = createReset.get();
+      await resetPasswordEmail(id, email, resetCode);
+      res.status(201).json({
+        status: 201,
+        message: 'Password reset link sent sucessfully. Please check your email!',
+        ResetPassword: createReset
+      });
+    } catch (error) {
+      return res.status(401).json({ message: 'Please try again' });
```

Comment:
I suggest this status code to be something else not 401 for example 520 for unknown error

Reply:
I think it is a good idea!